src/jswrap_object.c

```diff
@@ -222,8 +222,20 @@ void jswrap_object_keys_or_property_names_cb(
 
     while (symbols) {
       unsigned int i;
-      for (i=0;i<symbols->symbolCount;i++) {
-        JsVar *name = jsvNewFromString(&symbols->symbolChars[symbols->symbols[i].strOffset]);
+      unsigned char symbolCount = READ_FLASH_UINT8(&symbols->symbolCount);
+      unsigned short strOffset = READ_FLASH_UINT16(&symbols->symbols[i].strOffset);
+      for (i=0;i<symbolCount;i++) {
+#ifndef USE_FLASH_MEMORY
+        JsVar *name = jsvNewFromString(&symbols->symbolChars[strOffset]);
```

Comment:
Maybe we should just make jsvNewFromString work directly from flash with READ_FLASH_UINT8? It's used in a few places with constant strings, so would let us move quite a lot of extra stuff into flash?